doc/api/process.md

```diff
@@ -706,14 +706,15 @@ console.log(process.env.test);
 // => 1
 ```
 
-## process.emitWarning(warning[, name][, ctor])
+## process.emitWarning(warning[, type[, code]][, ctor])
```

Comment:
It's not just implied, it's explicitly stated in the arguments list that `warning` can be a string or an `Error`, and an example of passing an `Error` is shown immediately following the `If warning is passed as an Error object...` sentence. Immediately following the example is a clear statement that a `TypeError` will be thrown is `warning` is anything other than a string or `Error` object. The docs seem pretty explicit about it.